### ISSUE TITLE
Наш файл databoom.js еще и автоматически подключает jquery. Это приводит к ошибкам (так как jquery уже конечно был подключен).

### ISSUE THREAD

**Issue by vsalnikov** [Databoom/work#121]
Наш файл databoom.js еще и автоматически подключает jquery. Это приводит к ошибкам (так как jquery уже конечно был подключен).

А прикол еще и в том, что если jquery убрать (надеясь, что databoom.js его подключит, то тогда не работают другие библиотеки, использующие jquery)

**Comment by ixdu** [comment]
Дело в том, что есть уже как минимум 3и версии библиотеки и та версия что в корне лежит видимо стара. Вот появился повод заняться этим и почистить.
Вот в этих
https://databoom.space/libs/databoom.js
https://databoom.space/incl/databoom.js
jQuery нет

**Comment by ixdu** [comment]
Эта библиотека теперь основная, любые изменения или багофиксы будут в ней.
https://databoom.space/libs/databoom.js
Стара, доступная из корня останется для совместимости, но к использованию в новом не рекомендуется.

**Comment by vsalnikov** [comment]
А вот здесь (http://databoom.space/docs.html#!/jquery/part1) пишут другой адрес у библиотеки https://databoom.space/databoom.js

**Comment by ixdu** [comment]
Да, потому что документация ещё не обновлена, это с одной стороны. С другой на такое вот встроенное jquery возможно что-то завязано, выясним, если нет зависимостей то потихоньку сделаем одну вместо трёх)
